<dec f='include/python3.8/pyerrors.h' l='156' type='PyObject * PyErr_SetFromErrno(PyObject * )'/>
